src/components/SelectionList/RadioListItem.js

```diff
@@ -1,17 +1,39 @@
 import React from 'react';
 import {View} from 'react-native';
 import Text from '@components/Text';
+import Tooltip from '@components/Tooltip';
+import * as StyleUtils from '@styles/StyleUtils';
 import useThemeStyles from '@styles/useThemeStyles';
 import {radioListItemPropTypes} from './selectionListPropTypes';
 
-function RadioListItem({item, isFocused = false}) {
+function RadioListItem({item, showTooltip, textStyle, alternateTextStyle}) {
     const styles = useThemeStyles();
     return (
         <View style={[styles.flex1, styles.alignItemsStart]}>
-            <Text style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold]}>{item.text}</Text>
+            <Tooltip
+                shouldRender={showTooltip}
+                text={item.text}
+            >
+                <Text
+                    style={StyleUtils.combineStyles(textStyle, item.isSelected && styles.sidebarLinkTextBold)}
```

Comment:
The bold condition should be moved to the outer styles. Based on condition: `isUserItem || item.isSelected`